### PR TITLE
Hello Microverse Project - Pull Request

### PR DESCRIPTION
**Pull request by skabeo**
In this milestone I 
- added the HTML file with the text "Hello Microverse"
- I added a styles file to style the text.

### CODE REVIEW

**Comment by skabeo** [comment]
A request to add a html file with a header text and a simple css file styling the header

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @skabeo     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯 



Highlights
✅All linters are checked
✅ Descriptive commit messages 
  
Required Changes ♻️

- [x] Please consider naming your branches with descriptive names as regards your project. the `web-design-branch` you used does not ideally show what the changes you made did. in case of next time, please consider this 👍 

- [x] Your pull request was not used properly and was not descriptive enough. as seen in the snapshot below, it always uses the first field, and makes it as detailed as possible to explain all changes you made to the project. kindly effect this 👍 

## view:
image



- [x] Your README seems not to have been updated with the README template given. Please use this [Link](https://github.com/microverseinc/readme-template) to use the right one with all sections customized to your project 👍 

- [x] Somehow the changes you made to add all your linters are not showing in the files change. Which means linters are not running. please kindly fix this 👍 

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 1]
```diff
@@ -1 +1,55 @@
...
+
+👤 **Spencer Okyere**
+
+- GitHub: [@githubhandle](https://github.com/skabeo)
+- Twitter: [@twitterhandle](https://twitter.com/black_okyere)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/okyere-spencer-9b602623b)
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (aamirkhan2478) [inline README.md]: - [ ] Your MIT link is not working kindly fix this link.

File: README.md [hunk 2]
```diff
@@ -1 +1,55 @@
...
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```
Comment (aamirkhan2478) [inline README.md]: - [ ] Kindly remove this markdown code we don't need this.

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @skabeo , 



Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Highlights
✔️ Good Effort
✔️ All linters are checked
✔️ Descriptive commit messages

 ## Required Changes ♻️
Screenshot from 2023-02-16 19-27-45

- [ ] Please add the `build with` section to the README file. and remove the server and database and add HTML and CSS in side the client section.

Screenshot from 2023-02-16 19-27-58

- [ ] Please add the `Key Feature` section also and add your key feature here.

Screenshot from 2023-02-16 19-28-07

- [ ] Please add a `Live Demo` section and leave it as it is for this project but in future, you can use this link when you will live your website.

Screenshot from 2023-02-16 19-28-55

- [ ] Please add your social media handles then if someone clicks on your social media handle link it redirects to the specific social media website.

Screenshot from 2023-02-16 19-29-08

- [ ] Please add your `Future Feature` section also and add some future features which are you implemented in future.

Screenshot from 2023-02-16 19-29-18

- [ ] Please add `Contributing`, `Show your support` and  `Acknowledgment` sections.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shisui6 — APPROVED** [review]
Hi @skabeo , 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Descriptive PR title and summary ✅
- Follows Github flow ✅
- Good commit messages ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**